feat: Add the ability to cycle through panels (hidden configurable)

seppulcro opened the pull request:
Hi, I've added the ability to cycle through panels infinitely, with an option do go through hidden panels as well.

Solves #686 

[output.webm](https://github.com/user-attachments/assets/376afca2-69b8-41ce-a257-dfd0d1d3f3bd)

seppulcro committed:
feat: Add the ability to cycle through panels (hidden configurable)

ChilledMoth commented:
Nice, but why are you taking over the default hotkey for 'Open a new tab after the active tab'?

Just leave it undefined or set a new one, idk, 'Alt+M' or something (since 'Alt-Comma' and 'Alt+Period' are the defaults for 'Switch to previous panel' and 'Switch to next panel', respectively).

seppulcro committed:
fix: Change default keymap (overlapping), add backwards/forwards options

seppulcro commented:
> Nice, but why are you taking over the default hotkey for 'Open a new tab after the active tab'?
> 
> Just leave it undefined or set a new one, idk, 'Alt+M' or something (since 'Alt-Comma' and 'Alt+Period' are the defaults for 'Switch to previous panel' and 'Switch to next panel', respectively).

Whoops sorry I've been using that since I've setup it up and must've forgot to change it back.

I've also noticed a typo that I've fixed, and since I started using it for myself yesterday the ability to go only forwards didn't seem enough... so I added the ability to go backwards too in the latest commit.

seppulcro commented:
@ChilledMoth please re-validate when you have the time so I know if it's up to spec now 😁

llc0930 commented:
I personally think this feature should be added as an optional setting rather than replacing the original setting.

mbnuqw committed:
Merge branch 'v5' into feat/loop-cycle-panels

mbnuqw commented:
Thanks
